code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -529,7 +529,8 @@
 		prescriptions -= H.equipment_prescription
 	return Clamp(prescriptions, 0, 7)
 
-/datum/species/proc/handle_sprint_cost(var/mob/living/carbon/human/H, var/cost)
+// if calculation is true, it won't sap resources
+/datum/species/proc/handle_sprint_cost(var/mob/living/carbon/human/H, var/cost, var/calculation, var/post_calculation)
```

Comment:
These variable names are the most undescriptive thing ever. Please find some better names-

Reply:
It might be worth to add a doc-block at the top aswell